.github/workflows/build-strict.yml

```diff
@@ -45,5 +41,73 @@ jobs:
           rm -rf .lake/packages/proofwidgets/.lake/build/ir
           lake build ProofWidgets
 
+      - name: confirm all modules are included
+        run: lake exe mk_all --check
+
```

Comment:
The cache doesn't include the mk_all stuff, so it might as well be run before getting the cache so we can fail early.